### PR TITLE
Start Footer component 

### PR DESCRIPTION
**Pull request by michellewong793**
image

Starts footer component. Later work and improvements: Make responsive to smaller screen sizes, and mobile version

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
         <div css={footer}>
-            <p>Designed and Built by cLabs</p>
+            <div css={socialIcons}>
+                <img alt="twitter" src="/twitter.png" />
+                <img alt="signal" src="/signal.png" />
+                <img alt="telegram" src="/telegram.png" />
+                <img alt="reddit" src="/reddit.png" />
+            </div>
+            <div css={footerLinks}>
+                <a css={(theme) => [link, { color: theme.linkColor }]} href="/">FAQ</a>
```
Comment (aaronmgdr) [inline src/components/Footer.tsx:15]: if you use the `useTheme` hook then you dont need to do the `(theme) => function`  too. there are alternatives. 

if `useHook` its just `css={css(link, { color: theme.linkColor })}` 

if if no useHook exactly like it is here

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
 }
 
+
+const link = css({
+    textDecoration: "none",
+    fontFamily: "Inter",
+    fontWeight: 600
+})
+const socialIcons = css({ display: "flex", flexDirection: "row", justifyContent: "space-between", width: "10rem" })
+const footerLinks = css({ display: "flex", flexDirection: "row", justifyContent: "space-around", width: "19rem" })
```
Comment (aaronmgdr) [inline src/components/Footer.tsx:30]: wow thats a lot of `rem`

File: src/components/Footer.tsx (hunk 1 above)
Comment (michellewong793) [reply src/components/Footer.tsx]: i see! made an update

File: src/components/Footer.tsx (hunk 2 above)
Comment (michellewong793) [reply src/components/Footer.tsx:30]: haha ya i can always make it smaller, or to use another specific px

Comment (aaronmgdr) [reply src/components/Footer.tsx:30]: rems are cool, is there a specific reason for this size  or just it works?

**Review by aaronmgdr — APPROVED** [review]

File: src/components/Footer.tsx (hunk 2 above)
Comment (aaronmgdr) [reply src/components/Footer.tsx:30]: ps question here is informational. merge at your desire

Comment (michellewong793) [reply src/components/Footer.tsx:30]: it just works for now to contain the links, but we can change it to be specific if needed

**Pull request merged**